pubspec.yaml

```
@@ -10,12 +10,14 @@ environment:
 dependencies:
   collection: ^1.19.1
   csv: ^6.0.0
+  freezed_annotation: ^3.1.0
   http: ^1.4.0
-  json_annotation: ^4.8.1
+  json_annotation: ^4.9.0
   # path: ^1.8.0 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)